DOCUMENTATION.md

```diff
@@ -271,6 +275,66 @@ with open('file.csv', 'rb') as f:
     }
 ```
 
+#### Ask recipients to confirm their email address before they can download the file
+
+This feature is currently opt-in only. From 29 March 2023 it will apply to all new files by default, unless you opt out.
+
+##### Opt in
+
+To ask recipients to confirm their email address, set the `verify_email_before_download` flag to `true`.
+
+You will not need to do this after 29 March 2023.
```

Comment:
```suggestion
You can turn this feature on on a per-file basis. You will not need to do this after 29 March 2023.

To make the recipient confirm their email address before downloading the file, set the `verify_email_before_download` flag to `true`.
```

Reply:
Do we drop off the '2023' second time around just because it's too repetitive?